src/config/emailContent.js

```diff
@@ -0,0 +1,27 @@
+const CONTACT_MESSAGE_FIELDS = {
```

Comment:
It's neat that you have thought and taken care as to how email headings will appear in patient's email.